### DISCUSSION TITLE
Offset reference to another string

### DISCUSSION THREAD

**Discussion by nordstranger** [WerWolv/ImHex#1032] [Get Help]
Hi! Is there any way to match second element that have known offset from first hex pattern. But the address of first pattern is unknown, so I want to find it and then make offset to second address. The first pattern can exists in file few times. The second value is not pattern, it can be any u32.
I tried YARA etc. but don't understand if it possible.

**Comment by WerWolv** [comment]
Hey
I'm not quite sure I understand your question. Could you make an example maybe?

If you mean that you have one type that contains the address to another type inside, you can use pointers or the placement syntax inside of your first type

**Comment by nordstranger** [comment]
Example what i want to automate:
1) open hex file, find the second match result of  string "A3 87 4B 1D"
2) now I need to move forward for 8 bytes starting from "87", and that will be the offset I need. I want to highlight it in file.
